test/functional/apps/kibana_overview/_analytics.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { WebElementWrapper } from '../../services/lib/web_element_wrapper';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
```

Comment:
nit: I'd love a description of what's being tested for those in the future who might need to adjust this test, and comments throughout.  `describe` does a fairly decent job, but by its nature it's terse.

It's not something we do often at Elastic, but I'd love to set the precedent.

Reply:
I think functional tests are already quite descriptive, they are almost like a testing scenario: click here, wait for this to appear, make sure something is displayed on the page. I believe all of that can be read from the code with proper variable naming and good use of "describe", so I wouldn't go too crazy on commenting and documenting them.